bundles/org.openhab.binding.miele/src/main/resources/OH-INF/thing/dishwasher.xml

```diff
@@ -31,6 +31,15 @@
 				<description>The identifier identifies one certain appliance on the ZigBee network.</description>
 				<required>true</required>
 			</parameter>
+			<parameter name="protocol" type="text">
+                <label>Protocol</label>
```

Comment:
Could you please reapply the formatter here (and on all other config files)? The indentation looks broken...

Reply:
@kaikreuzer Indeed, not sure what happened here